npm run firefox does not run Firefox

```sh
$ git rev-parse --short HEAD # c138fbb
```

---

### Steps to reproduce:

```sh
$ npm run firefox

> tracking-protection-experiment@1.0.0 firefox /Users/pdehaan/dev/github/mozilla/blok
> npm run bundle && web-ext run


> tracking-protection-experiment@1.0.0 bundle /Users/pdehaan/dev/github/mozilla/blok
> browserify -d -t envify src/js/background.js > src/js/background.bundle.js

Running web extension from /Users/pdehaan/dev/github/mozilla/blok

run: InvalidManifest: Error parsing manifest.json at /Users/pdehaan/dev/github/mozilla/blok/manifest.json: InvalidManifest: Could not read manifest.json file at /Users/pdehaan/dev/github/mozilla/blok/manifest.json: Error: ENOENT: no such file or directory, open '/Users/pdehaan/dev/github/mozilla/blok/manifest.json'
    at /Users/pdehaan/dev/github/mozilla/blok/node_modules/web-ext/dist/webpack:/src/util/manifest.js:9:1805


npm ERR! Darwin 15.6.0
npm ERR! argv "/Users/pdehaan/.nvm/versions/node/v6.3.0/bin/node" "/Users/pdehaan/.nvm/versions/node/v6.3.0/bin/npm" "run" "firefox"
npm ERR! node v6.3.0
npm ERR! npm  v3.10.3
npm ERR! code ELIFECYCLE
npm ERR! tracking-protection-experiment@1.0.0 firefox: `npm run bundle && web-ext run`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the tracking-protection-experiment@1.0.0 firefox script 'npm run bundle && web-ext run'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the tracking-protection-experiment package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run bundle && web-ext run
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs tracking-protection-experiment
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls tracking-protection-experiment
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/pdehaan/dev/github/mozilla/blok/npm-debug.log
```

I was in the middle of trying to test #156, but I'll try and debug more later. I'm guessing I may just have to pass in the `src` directory now that this was refactorized.

--- a/package.json
+++ b/package.json
@@ -9,10 +9,10 @@
   "scripts": {
     "bundle": "browserify -d -t envify src/js/background.js > src/js/background.bundle.js",
     "build": "npm test && MODE=production npm run bundle && web-ext build -s src",
-    "build-dev": "npm test && MODE=dev npm run bundle && web-ext build",
+    "build-dev": "npm test && MODE=dev npm run bundle && web-ext build -s src",
     "coverage": "nyc npm test",
     "coveralls": "nyc npm test && nyc report --reporter=text-lcov | coveralls",
-    "firefox": "npm run bundle && web-ext run",
+    "firefox": "npm run bundle && web-ext run -s src",
     "lint": "standard && web-ext lint --self-hosted -s src",
     "test": "tape tests/**/*.js | faucet",
     "watch": "npm-watch"

